### PR TITLE
MUAN-8089:feat:Change the background with the following picture

### PR DESCRIPTION
**Pull request by beto-dt**
<h1>UI</h1>
Screen Shot 2022-06-09 at 1 58 09 AM
<h1>COVERAGE</h1>
Screen Shot 2022-06-09 at 2 01 11 AM

### CODE REVIEW

File: mushroomsoft/app/src/app/components/technology/technology.component.html [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
-                    </div>
-                </div>
-            </div>
+  <div class="ms-container-3 ms-w-container">
+    <div class="ms-layout-grid">
+      <h1 id="tecnology" class="ms-heading-8 ms-tecnology-title">
+      {{titleTechnology}}
+      </h1>
+      <p id="ms-tecnology-description" class="ms-paragraph-18">
+      <br />{{descriptionTechnology}} <br />
```
Comment (jogaec22) [inline mushroomsoft/app/src/app/components/technology/technology.component.html:8]: Delete spaces in <br` `/>, would be <br/>

Comment (beto-dt) [reply mushroomsoft/app/src/app/components/technology/technology.component.html]: resolved

Comment (jogaec22) [reply mushroomsoft/app/src/app/components/technology/technology.component.html]: in all occurrences, please.

File: mushroomsoft/app/src/index.html [hunk 2]
```diff
@@ -5,9 +5,9 @@
   <title>MushroomSoft IT</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link href="/assets/favicon/favicon.png" rel="shortcut icon" type="image/x-icon">
+  <link href="assets/favicon/favicon.png" rel="shortcut icon" type="image/x-icon">
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:100,100italic,200,200italic,300,300italic,400,400italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic" media="all">
-  <link href="/assets/favicon/favicon.png" rel="apple-touch-icon">
+  <link href="assets/favicon/favicon.png" rel="apple-touch-icon">
```
Comment (leoriofrio) [inline mushroomsoft/app/src/index.html:10]: use routerLink

File: mushroomsoft/app/src/app/components/service/service.component.html [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
-                    <h1 class="ms-heading-17">{{items.title}}</h1>
-                    <p class="ms-paragraph-11">
-                    {{items.description}}<br />
-                    </p>
-                </div>
-            </div>
-        </div>
-    </div>
+  <div class="ms-w-container">
+          <h1 id="services" class="{{serviceClass}}">{{serviceTitle}}</h1>
```
Comment (leoriofrio) [inline mushroomsoft/app/src/app/components/service/service.component.html:3]: fix tabs space

**Review by leoriofrio — CHANGES_REQUESTED** [review]

File: mushroomsoft/app/src/app/components/technology/technology.component.html (hunk 1 above)
Comment (beto-dt) [reply mushroomsoft/app/src/app/components/technology/technology.component.html]: resolved

File: mushroomsoft/app/src/index.html (hunk 2 above)
Comment (beto-dt) [reply mushroomsoft/app/src/index.html:10]: But it is the index.html in those lines I only call the icon of the tab and the font, when I use routerLink the image and the font are no longer loaded

File: mushroomsoft/app/src/app/components/service/service.component.html (hunk 3 above)
Comment (beto-dt) [reply mushroomsoft/app/src/app/components/service/service.component.html]: reoslved

**Comment by jogaec22** [comment]
> Please, @beto-dt can you review these practices to implement the CSS to apply in this refactor https://learn.shayhowe.com/advanced-html-css/responsive-web-design/#mobile-first https://aukera.es/blog/diseno-responsive/ Maybe with this, we can reduce the file size by avoiding duplicating styles like this: image it is necessary that you apply these best practices where you implement mobile styles first y optimize the code lines and don't need to redefine styles again.

I agree you must apply the "First mobile" principle.

File: mushroomsoft/app/src/index.html (hunk 2 above)
Comment (leoriofrio) [reply mushroomsoft/app/src/index.html:10]: you're right

**Review by leoriofrio — APPROVED** [review]

**Pull request merged**